hedera-node/hedera-app/src/test/resources/bootstrap.properties

```diff
@@ -220,7 +220,7 @@ hedera.prefetch.codeCacheTtlSecs=600
 utilPrng.isEnabled=true
 tokens.autoCreations.isEnabled=true
 accounts.blocklist.enabled=true
-accounts.blocklist.resource=evm-addresses-blocklist.csv
+accounts.blocklist.path=evm-addresses-blocklist.csv
```

Comment:
nit : Since we changed it to path, we could probably mention path here to avoid confusion

Reply:
Fixed.